rust/main/agents/relayer/src/msg/op_batch.rs

```diff
@@ -44,12 +44,28 @@ impl OperationBatch {
         };
 
         if !excluded_ops.is_empty() {
-            warn!(excluded_ops=?excluded_ops, "Either operations reverted in the batch or the txid wasn't included. Sending them back to prepare queue.");
-            let reason = ReprepareReason::ErrorSubmitting;
-            let status = Some(PendingOperationStatus::Retry(reason.clone()));
-            for mut op in excluded_ops.into_iter() {
-                op.on_reprepare(None, reason.clone());
-                prepare_queue.push(op, status.clone()).await;
+            let Some(first_item) = excluded_ops.first() else {
```

Comment:
let's combine these two in a pattern match

Reply:
Done